app/src/main/java/org/simple/clinic/newentry/country/FormFieldFactory.kt

```diff
@@ -2,30 +2,65 @@ package org.simple.clinic.newentry.country
 
 import org.simple.clinic.appconfig.Country
 import org.simple.clinic.newentry.form.AgeField
+import org.simple.clinic.newentry.form.BusinessIdentifierField
 import org.simple.clinic.newentry.form.DateOfBirthField
 import org.simple.clinic.newentry.form.DistrictField
 import org.simple.clinic.newentry.form.GenderField
 import org.simple.clinic.newentry.form.InputField
 import org.simple.clinic.newentry.form.LandlineOrMobileField
 import org.simple.clinic.newentry.form.PatientNameField
 import org.simple.clinic.newentry.form.StateField
+import org.simple.clinic.newentry.form.StreetAddressField
 import org.simple.clinic.newentry.form.VillageOrColonyField
+import org.simple.clinic.newentry.form.ZoneField
 import org.threeten.bp.LocalDate
 import org.threeten.bp.format.DateTimeFormatter
 
+// TODO 1. Convert to class and take in the date formatter and today, just pass in the country object.
+// TODO 2. Rename to InputFieldsFactory.
 object FormFieldFactory {
   fun fields(
       dateTimeFormatter: DateTimeFormatter,
       today: LocalDate,
       country: Country
+  ): List<InputField<*>> {
+    return when (country.isoCountryCode) {
+      "IN" -> formFieldsForIndia(dateTimeFormatter, today)
```

Comment:
In the current `master`, we have constants in the `Country` class to check the country codes. We should use those instead here.

Reply:
Rebased and fixed it. Done.